### DISCUSSION TITLE
AM I installing incorrectly or missing dependencies? I get this error when trying to use dotfiles

### DISCUSSION THREAD

**Discussion by adelwolf** [zDyant/HyprNova#12] [General]
![image](https://github.com/user-attachments/assets/a2819f5e-07d8-491e-a96d-9c2744b93da0)

**Comment by zDyant** [comment]
These are outdated options, removing them should fix it

**Reply by adelwolf** [reply]
awesome, thank you! ive also noticed that the waybar theme selector just removes my waybar anytime i try to change it from default. any advice on what could be wrong with that?

edit: need to log out and back in every time i change it for it to work.
also, wallpaper selector doesn't save my selections between logins. any way to fix that?

**Reply by zDyant** [reply]
> awesome, thank you! ive also noticed that the waybar theme selector just removes my waybar anytime i try to change it from default. any advice on what could be wrong with that?

Not a clue, never happen to me

> also, wallpaper selector doesn't save my selections between logins. any way to fix that?

Odd, try to run the script on the terminal and show me the log
